anta/tests/connectivity.py

```diff
@@ -77,14 +74,19 @@ def test(self) -> None:
 
 
 class VerifyLLDPNeighbors(AntaTest):
-    """Verifies that the provided LLDP neighbors are present and connected with the correct configuration.
+    """Verifies the connection status of the specified LLDP (Link Layer Discovery Protocol) neighbors.
+
+    This test performs the following checks for each specified LLDP neighbor:
+
+      1. Confirming matching ports on both local and neighboring devices.
```

Comment:
while we are refactoring I would add somewhere that if the given info is. match potentially we dont raise if any extra neighbor is detected on a given interfacer

e.g.

input Ethernet1 connected to Foo on Ethernet2

if connected to Foo on both Ethernet2 and Ethernet3 then this is accepted

Reply:
Added docstring.